lib/matplotlib/axes/_base.py

```diff
@@ -2613,7 +2613,7 @@ def _update_title_position(self, renderer):
     # Drawing
     @martist.allow_rasterization
     def draw(self, renderer=None, inframe=False):
```

Comment:
What about the parameters here? They're undocumented (though they were before too).